ironfish/src/wallet/walletdb/decryptedNoteValue.ts

```diff
@@ -14,17 +14,22 @@ export interface DecryptedNoteValue {
   // These fields are populated once the note's transaction is on the main chain
   index: number | null
   nullifier: Buffer | null
+  blockHash: Buffer | null
+  sequence: number | null
```

Comment:
should we do blockSequence, or no?

Reply:
On one hand, blockSequence is clearer, but, on the other hand, TransactionValue uses sequence

Do we choose clarity or consistency?

Reply:
I think I prefer `sequence` since there are only a couple of places in the code where we currently use `blockSequence`, but I don't have a strong preference!